explorer/syncstatus.go

```diff
@@ -71,11 +71,6 @@ func (exp *explorerUI) BeginSyncStatusUpdates(barLoad chan *dbtypes.ProgressBarL
 	go func() {
 		timer := time.NewTicker(syncStatusInterval)
 		for range timer.C {
-			if barLoad == nil {
-				log.Debug("Stopping progress bar signals.")
-				timer.Stop()
```

Comment:
Another solution to avoid leaking this goroutine is needed.  Note that the other goroutine below nils `barLoad`.

Reply:
Maybe just stop the timer in the other goroutine. ofc would need to capture the timer by both goroutines.

Reply:
Since the timer never stops the goroutine will always be running in the background.

Reply:
pushed a revision.